.github/workflows/validate.yml

```diff
@@ -20,6 +20,17 @@ jobs:
         with:
           fetch-depth: 0
 
+      # Be careful! We need this action to be able to make comments on PRs from external repos, but
+      # doing so requires using the pull_request_target workflow trigger. Using this trigger will
+      # run the workflow in the context of the base branch, not the remote branch, which gives the
+      # workflow access to secrets. You MUST NOT clone any remote code to execute here. Since we're
+      # only cloning data files and these files are never executed (only parsed), this step is OK.
+      - name: Checkout remote data files
+        run: |
+          git remote add pr ${{ github.event.pull_request.head.repo.clone_url }}
+          git fetch pr
+          git checkout pr/${{ github.event.pull_request.head.ref }} -- data/
```

Comment:
Nit: this isn't entirely true:
> Since we're only cloning data files

Technically we could be cloning _any_ kinds of files in the `data/` dir of the fork. The possible risk I see is if anything in CI would automatically detect and run based on something in here, which could even include json, yaml etc. 

Since it looks like the only CI runner on this repo is GHA, which [only accepts config ](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#about-yaml-syntax-for-workflows) from `.github/workflows`, I don't see a clear path to smuggling something in.
That said, it might make sense to throw an error before the action runs if there are any files that don't match expectations.

Reply:
I've added logic to guarantee that we are only downloading either data.json, logo.svg, or logo.png files.